pennylane/transforms/op_transforms.py

```diff
@@ -0,0 +1,481 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the @op_transform decorator.
+"""
+# pylint: disable=protected-access
+import functools
+import inspect
+import os
+import warnings
+
+import pennylane as qml
+
+
+class OperationTransformError(Exception):
+    """Raised when there is an error with op_transform logic"""
+
+
+def _make_tape(obj, wire_order, *args, **kwargs):

```

Comment:
How come this is not a method like `_make_wrapper`? Say a staticmethod.

Having the standard docstring sections (Args, Returns, etc.) could be helpful even though this is a private function.

Reply:
Good idea!

Reply:
Done (locally)